### ISSUE TITLE
MSSIM loss function

### ISSUE THREAD

**Issue by Dpw506** [liuh127/Two-branch-dehazing#3]
Hi, Thank you for this work. I ran into the following problems while running the code you provided that is the MSSIM loss function you provided missing 1-MSSIM? What I see in the code is -MSSIM. How should I understand this? Looking forward to your reply.

**Comment by liuh127** [comment]
Hi， please see the link for detailed implementation of MS-SSIM loss.
https://github.com/liuh127/Two-branch-dehazing/blob/7fa08e810dba73088081384ec4a1940c872784e2/pytorch_msssim/__init__.py#L124

We use the implementation in https://github.com/jorge-pessoa/pytorch-msssim.

**Comment by liuh127** [comment]
> Hi, Thank you for this work. I ran into the following problems while running the code you provided that is the MSSIM loss function you provided missing 1-MSSIM? What I see in the code is -MSSIM. How should I understand this? Looking forward to your reply.

Hi， please see the link for detailed implementation of MS-SSIM loss.
https://github.com/liuh127/Two-branch-dehazing/blob/7fa08e810dba73088081384ec4a1940c872784e2/pytorch_msssim/__init__.py#L124

We use the implementation in https://github.com/jorge-pessoa/pytorch-msssim.

**Comment by Dpw506** [comment]
Thank you very much!
